src/@types/type-modules/svg.ts

```diff
@@ -4,7 +4,7 @@ import { StringOrNumber } from './generic';
 export type StrokeLinecap = keyof typeof LineCap;
 export type StrokeLinejoin = keyof typeof LineJoin;
 
-export type SVGProperties = 'fillColor' | 'fillOpacity' | 'strokeColor' | 'strokeOpacity' | 'strokeWidth';
+export type SVGProperties = 'fillColor' | 'fillOpacity' | 'strokeColor' | 'strokeOpacity' | 'strokeWidth' | 'className';
```

Comment:
The name of the type is confusing. The `className` property should not be here. This type is used only for properties that are animatable.